cmake/FindHDK.cmake

```diff
@@ -217,21 +217,12 @@ IF (HDK_FOUND)
 	  SET ( HDK_DEFINITIONS " -DDLLEXPORT= -D_GNU_SOURCE -DLINUX -DAMD64 -m64 -fPIC -DSIZEOF_VOID_P=8 -DSESI_LITTLE_ENDIAN -DENABLE_THREADS -DUSE_PTHREADS -D_REENTRANT -D_FILE_OFFSET_BITS=64 -c -DGCC4 -DGCC3 -Wno-deprecated")
 	  SET ( DEBUG_HDK_DEFINITIONS " -DDLLEXPORT= -D_GNU_SOURCE -DLINUX -DAMD64 -m64 -fPIC -DSIZEOF_VOID_P=8 -DSESI_LITTLE_ENDIAN -DENABLE_THREADS -DUSE_PTHREADS -D_REENTRANT -D_FILE_OFFSET_BITS=64 -c -DGCC4 -DGCC3 -Wno-deprecated -g")
 	ENDIF ()
-
-    # Split each definition and add them separately (otherwise we can get into weird string manipulation issues within CMake)
-    SEPARATE_ARGUMENTS( DEBUG_HDK_DEFINITIONS )
-    SEPARATE_ARGUMENTS( HDK_DEFINITIONS )
-
     # MESSAGE ( "DEBUG_HDK_DEFINITIONS = ${DEBUG_HDK_DEFINITIONS}" )
     # MESSAGE ( "HDK_DEFINITIONS = ${HDK_DEFINITIONS} ")
     IF (CMAKE_BUILD_TYPE MATCHES Debug)
-      FOREACH ( DEFINITION ${DEBUG_HDK_DEFINITIONS} )
-        ADD_DEFINITIONS( ${DEFINITION} )
-      ENDFOREACH ()
+      ADD_DEFINITIONS ( "${DEBUG_HDK_DEFINITIONS}" )
     ELSE (CMAKE_BUILD_TYPE MATCHES Debug)
-      FOREACH ( DEFINITION ${HDK_DEFINITIONS} )
-        ADD_DEFINITIONS( ${DEFINITION} )
-      ENDFOREACH ()
+      ADD_DEFINITIONS ( "${HDK_DEFINITIONS}" )
```

Comment:
Actually wait - I added these because this CMake build was failing for Houdini 17. Why have these been removed? I need to double check if this still works

Reply:
Yep, removing this still breaks for me

Reply:
This patch only removes whitespace and shouldn't have changed any code. I suspect another pull request was merged before this one causing the code merge result to be different. I'll rebase the patch today and see if it resolves itself.

Reply:
Fixed in my latest update.